clinvar_example_750_rows.tsv

```diff
@@ -1,750 +1,750 @@
-1	2160499	C	T	ALT	213681	SKI	Benign	0	1	0	criteria provided, single submitter	1	NM_003036.3:c.294C>T	NP_003027.1:p.Thr98=	GeneDx	not specified;not specified						germline	MedGen:CN169374
-1	2160517	A	T	ALT	227947	SKI	Likely benign	0	1	0	criteria provided, single submitter	1	NM_003036.3:c.312A>T	NP_003027.1:p.Val104=	Laboratory for Molecular Medicine,Partners HealthCare Personalized Medicine	not specified;Not specified	24033266					germline	MedGen:CN169374
-1	2160552	G	A	ALT	37258	SKI	Pathogenic	1	0	0	criteria provided, single submitter	1	NM_003036.3:c.347G>A	NP_003027.1:p.Gly116Glu	OMIM;Baylor-Hopkins Center for Mendelian Genomics,Johns Hopkins University	Shprintzen-Goldberg syndrome;SHPRINTZEN-GOLDBERG CRANIOSYNOSTOSIS SYNDROME	20301454,23023332		Infancy	<1 / 1 000 000		de novo;germline	GeneReviews:NBK1277;Genetic Alliance:Shprintzen-Goldberg+craniosynostosis+syndrome/6581;Genetics Home Reference:shprintzen-goldberg-syndrome;MedGen:C1321551;OMIM:182212;Office of Rare Diseases:4861;Orphanet:2462;SNOMED CT:83092002
-1	2160554	G	C	ALT	224868	SKI	Pathogenic	1	0	0	criteria provided, single submitter	1	NM_003036.3:c.349G>C	NP_003027.1:p.Gly117Arg	Baylor-Hopkins Center for Mendelian Genomics,Johns Hopkins University	Shprintzen-Goldberg syndrome	20301454,23023332		Infancy	<1 / 1 000 000		de novo	GeneReviews:NBK1277;Genetic Alliance:Shprintzen-Goldberg+craniosynostosis+syndrome/6581;Genetics Home Reference:shprintzen-goldberg-syndrome;MedGen:C1321551;OMIM:182212;Office of Rare Diseases:4861;Orphanet:2462;SNOMED CT:83092002
-1	2160557	G	A	ALT	222819	SKI	Likely pathogenic	1	0	0	criteria provided, single submitter	1	NM_003036.3:c.352G>A	NP_003027.1:p.Glu118Lys	Blueprint Genetics	Shprintzen-Goldberg syndrome	20301454		Infancy	<1 / 1 000 000		germline	GeneReviews:NBK1277;Genetic Alliance:Shprintzen-Goldberg+craniosynostosis+syndrome/6581;Genetics Home Reference:shprintzen-goldberg-syndrome;MedGen:C1321551;OMIM:182212;Office of Rare Diseases:4861;Orphanet:2462;SNOMED CT:83092002
-1	2160565	C	T	ALT	213682	SKI	Benign	0	1	0	criteria provided, single submitter	1	NM_003036.3:c.360C>T	NP_003027.1:p.Arg120=	GeneDx	not specified;not specified						germline	MedGen:CN169374
-1	2160622	C	T	ALT	213683	SKI	Benign	0	1	0	criteria provided, multiple submitters, no conflicts	2	NM_003036.3:c.417C>T	NP_003027.1:p.Ile139=	GeneDx;Invitae	not specified;Shprintzen-Goldberg syndrome	20301454		Infancy	<1 / 1 000 000		germline	OMIM:182212;GeneReviews:NBK1277;MedGen:C1321551;Office of Rare Diseases:4861;Genetic Alliance:Shprintzen-Goldberg+craniosynostosis+syndrome/6581;Orphanet:2462;MedGen:CN169374;SNOMED CT:8309200
```

Comment:
For molecular_consequence, it may have more than one values depending on the transcript. 
e.g.  [https://www.ncbi.nlm.nih.gov/clinvar/RCV000170331/](url)
Thus I prefer to keep this way of recording.